tests/functional/docs/test_dynamodb.py

```diff
@@ -42,19 +42,19 @@ def test_document_interface_is_documented(self):
         request_syntax_contents = self.get_request_syntax_document_block(
             method_contents)
         self.assert_contains_lines_in_order([
-            '        response = table.put_item(',
```

Comment:
So the way our assertion / search works here is that it skips through whitespace to find the selected strings. Asserting whitespace is brittle because we may want to tweak formatting. In this case, the custom methods were being put in the wrong section of the generated docs, and moving them to the correct section changed the indentation somewhat.